test/integration/validate-analysisdefinition/analysisdef_bad.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: AnalysisDefinition
+metadata:
+  name: analysisdefinition-bad
+spec:
+  objectives:
+    - analysisValueTemplateRef:
+        name: response_time_p95
+        namespace: keptn-lifecycle-toolkit-system
+      targets:
+        failure:
+          - lessThan:
+              fixedValue: 600
+            greaterThan:
```

Comment:
please add a short comment explaining why applying this resource is expected to fail